src/test/java/core/basesyntax/db/service/impl/FileCsvReaderServiceImplTest.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.db.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.service.FileCsvReaderService;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileCsvReaderServiceImplTest {
+    private static final String VALID_FILE_PATH = "src/main/resources/input.csv";
+    private static final String INVALID_FILE_PATH = "invalidFilePath";
+    private static FileCsvReaderService fileCsvReaderService;
+
+    @BeforeClass
+    public static void setUp() {
+        fileCsvReaderService = new FileCsvReaderServiceImpl();
+    }
+
+    @Test
+    public void readFromFile_validFilePath_ok() {
+        List<String> expected = new ArrayList<>(List.of(
```

Comment:
```suggestion
        List<String> expected = List.of(
```
fix everywhere where you use such a code